src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -384,24 +384,7 @@ public string TransferEncoding
         }
 
 
-        public bool KeepAlive
-        {
-            get
-            {
-                return _webHeaderCollection[HttpKnownHeaderNames.KeepAlive] == bool.TrueString;
-            }
-            set
-            {
-                if (value)
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.TrueString);
-                }
-                else
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.FalseString);
-                }
-            }
-        } 
+        public bool KeepAlive { get; set; } = true;
```

Comment:
I don't understand this particular C# syntax:

```c#
public bool KeepAlive { get; set; } = true;
```

What does it mean to do the "= true;" part?